src/plots/polar/layout_defaults.js

```diff
@@ -201,7 +202,9 @@ function handleAxisTypeDefaults(axIn, axOut, coerce, subplotData, dataAttr) {
         }
 
         if(trace && trace[dataAttr]) {
-            axOut.type = autoType(trace[dataAttr], 'gregorian');
+            axOut.type = autoType(trace[dataAttr], 'gregorian', {
+                autotypenumbers: autotypenumbers
```

Comment:
while we're at it, should polar axes get `noMultiCategory` too?

Reply:
Added in https://github.com/plotly/plotly.js/pull/5240/commits/1198e56542ef03a342de5e100094bb5f6bdd3001.